packages/wdio-utils/src/driver/index.ts

```diff
@@ -107,6 +107,12 @@ export async function startWebDriver (options: Options.WebDriver) {
         /**
          * Firefox
          */
+        const { executablePath } = await setupPuppeteerBrowser(cacheDir, caps)
+        caps['moz:firefoxOptions'] = deepmerge(
+            { binary: executablePath },
+            caps['moz:firefoxOptions'] || {}
+        )
+        delete caps.browserVersion
```

Comment:
I understand `geckodriver` is very specific with the versions passsed to session creation requests, but won't we loose valuable information for all kinds of reporters and services by deleting the `browserVersion`?

FYI I worked around this by getting semver info and modifying the version like so:
```suggestion
    // Set proper version in capabilities
    // Geckodriver does not like 0-based patch versions
    caps.browserVersion = `${major}.${minor}${patch ? `.${patch}` : ''}`
```

Reply:
Good point, yeah would prefer to keep the version. I will look get this added.

Reply:
I actually looked into where we pass information through and it seems all reporter get the `browserVersion` from the capabilities object returned by the driver. Deleting them doesn't harm that.

Reply:
Apologies, seems like I forgot to re-build after switching. It was indeed related and have fixed the cause. I have also made it so that the edge capabilities are only added when on Windows.